consensus/polybft/state.go

```diff
@@ -181,6 +187,21 @@ func decodeEventData(eventDataMap map[string]interface{}, log *ethgo.Log,
 	return eventCreator(id, sender, receiver, data), nil
 }
 
+// convert converts types.Log to ethgo.Log
+func convert(log *types.Log) *ethgo.Log {
```

Comment:
How about calling it `convertLog` maybe? I find simply `convert` too generic name.

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/852/commits/2905c39f46bf6a7af6d323b1e016924bb55a3573